app/src/main/java/com/pajato/android/gamechat/fragment/ChatFragment.java

```diff
@@ -44,6 +48,7 @@
     @Override
     public void onCreateOptionsMenu(final Menu menu, final MenuInflater menuInflater) {
         menuInflater.inflate(R.menu.chat_menu, menu);
+        initFabListener();
```

Comment:
I'm having a little angst over initializing the FAB button here.  I have a hunch it belongs in the fragment initialization.  Also, fwiw, I tend to prefer explicit listeners rather than anonymous ones but I seem to be in a distinct minority.  I use anonymous listeners as a rare exception rather than as a rule.  So I would have written: 

fab.setMenuListener(new FabMenuHandler(...));  or something similar.